src/integrationCapture.ts

```diff
@@ -134,7 +134,7 @@ const integrationMapping: IntegrationIdMapping = {
 
     // Snapchat
     // https://businesshelp.snapchat.com/s/article/troubleshooting-click-id?language=en_US
-    ScCid: {
+    sccid: {
```

Comment:
If this generically allows any query parameter click id to be case insensitive, is there a reason for changing the casing here?  I'd expect the keys for the click ids to be the same.

Reply:
this is where we compare what to expect coming in the url after lowercasing them, if I leave this as ScCid and we lowercase every click id, it will always fail since it will never match it